addons/my-ocular/addon.json

```diff
@@ -1,73 +1,130 @@
 {
-  "name": "ocular integration",
-  "description": "Shows ocular statuses and forum post reactions. Uses my-ocular.jeffalo.net to obtain data.",
+  "name": "Ocular and Aviate Integration",
+  "description": "Shows ocular/aviate statuses and forum post reactions. Uses my-ocular.jeffalo.net to obtain data.",
   "credits": [
     {
-      "name": "Jeffalo"
+      "name": "Jeffalo",
+      "id": "ocularAddon",
+      "note": "Ocular addon",
+      "link": "https://github.com/jefalo"
+    },
+    {
+      "name": "MystPi",
+      "id": "aviateAddon",
+      "note": "Aviate addon",
+      "link": "https://github.com/MystPi"
+    },
+    {
+      "name": "Norbiros",
+      "id": "addonVersion",
+      "link": "https://github.com/Norbiros"
     }
   ],
   "userscripts": [
     {
       "url": "discuss.js",
-      "matches": ["https://scratch.mit.edu/discuss/topic/*"],
+      "matches": ["https://scratch.mit.edu/discuss/topic/*", "https://scratch.mit.edu/users/*"],
       "if": {
-        "settings": { "discuss": true }
```

Comment:
Why this diff?

Reply:
Code formatting (github action) changed that

Reply:
But it says that the action failed...

Reply:
I changed some things in github branch protection, and because of that, in some commits Prettier was broken, but it worked at this moment (and it works now).